src/sentry/api/event_search.py

```diff
@@ -836,6 +837,12 @@ def to_list(value):
     "p75": {"result_type": "duration", "aggregations": [["quantile(0.75)(duration)", None, "p75"]]},
     "p95": {"result_type": "duration", "aggregations": [["quantile(0.95)(duration)", None, "p95"]]},
     "p99": {"result_type": "duration", "aggregations": [["quantile(0.99)(duration)", None, "p99"]]},
+    "error_rate": {
+        "result_type": "number",
+        "aggregations": [
+            ["divide(countIf(notEquals(transaction_status, 0)), count(*))", None, "error_rate"]
```

Comment:
Have query pre-processor functions landed yet? It would be better if this was implemented in snuba.

Reply:
I figured that I could add this quickly here, in the same format as the existing functions, to meet the performance beta deadlines. Then when we rewrite all these to use snuba query pre processors it's just one more processor.